code/game/gamemodes/game_mode.dm

```diff
@@ -637,6 +637,16 @@
 		if(SSshuttle.emergency.is_hijacked())
 			SSticker.news_report = SHUTTLE_HIJACK
 
+/**
+  * Given a list of minds, returns TRUE if they completed all their objectives, FALSE otherwise
+  */
+/datum/game_mode/proc/didAntagsWin(list/antags)
```

Comment:
And this proc exists because....?

Reply:
because I based this off of the same branch as the 2 other PRs which actually needed that, didn't realize it wasn't needed for this till I had done it